cmd/otelcontribcol/main.go

```diff
@@ -33,13 +33,14 @@ func main() {
 		log.Fatalf("failed to build components: %v", err)
 	}
 
-	info := component.ApplicationStartInfo{
-		ExeName: "otelcontribcol",
-		Version: version.Version,
-		GitHash: version.GitHash,
+	info := component.BinaryInfo{
+		Command:     "otelcontribcol",
+		Description: "OpenTelemetry Collector Contrib",
+		Version:     version.Version,
+		GitHash:     version.GitHash,
```

Comment:
I think this was removed, you probably need to rebase

Reply:
Done and updated the PR